website/docs/sql-reference/clauses/sql-limit.md

```diff
@@ -0,0 +1,74 @@
+---
+id: limit
+title: SQL LIMIT
+description: Adding the LIMIT clause to a query will limit the number of rows returned.
+slug: /sql-reference/limit
+---
+
+<head>
+    <title>Working with the SQL LIMIT clause</title>
+</head>
+
+When you’re developing data models or drafting up a query, do you usually need to see all results from it? Not normally. Hence, we LIMIT.
+
+Adding the LIMIT clause to a query will limit the number of rows returned. It’s useful for when you’re developing data models, ensuring SQL in a query is functioning as expected, and wanting to save some money during development periods.
+
+## How to use the LIMIT clause in a query
+
+To limit the number of rows returned from a query, you would pass the LIMIT in the last line of the query with the number of rows you want returned:
+
+```sql
+select
+	some_rows
+from my_data_source
+limit <integer>
+```
+
+Let’s take a look at a practical example using LIMIT below.
+
+### LIMIT example
+
+```sql
+select
+	order_id,
+	order_date,
+	rank () over (order by order_date) as order_rnk
+from {{ ref('orders') }}
+order by 2
+limit 5
+```
+
+This simple query using the [Jaffle Shop’s](https://github.com/dbt-labs/jaffle_shop) `orders` table will return these exact 5 rows:
+
+| order_id | order_date | order_rnk |
+|:---:|:---:|:---:|
+| 1 | 2018-01-01 | 1 |
+| 2 | 2018-01-02 | 2 |
+| 3 | 2018-01-04 | 3 |
+| 4 | 2018-01-05 | 4 |
+| 5 | 2018-01-05 | 4 |
+
+After ensuring that this is the result you want from this query, you can omit the LIMIT in your final data model.
+
+:::tip Save money and time by limiting data in development
```

Comment:
You could just cut directly to noting the dbt macro (as you mention the cost reduction during development use cases in the opener).